src/app/upload/upload.component.spec.ts

```diff
@@ -24,7 +24,7 @@ import { MockBackend} from '@angular/http/testing';
 import { RouterTestingModule } from '@angular/router/testing';
 import { UploadComponent } from "./upload.component";
 import { UploadService } from "./shared/upload.service";
-import { FormsModule } from "@angular/forms";
+import {FormsModule, ReactiveFormsModule} from "@angular/forms";
```

Comment:
All imports should be with spaces and with this: ' ' quotes
import { Module } from 'path'